packages/web/src/components/createCell.tsx

```diff
@@ -127,15 +128,18 @@ export function createCell<CellProps = any>({
   Failure,
   Empty,
   Success,
+  displayName,
 }: CreateCellProps<CellProps>): React.FC<CellProps> {
   if (global.__REDWOOD__PRERENDERING) {
     // If its prerendering, render the Cell's Loading component
     // and exit early. The apolloclient loading props aren't available here, so 'any'
     return (props) => <Loading {...(props as any)} />
   }
 
-  return (props) => {
+  const NamedCell = (props: CellProps) => {
     // destructuring to not pass children to beforeQuery
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
I dunno man.... this looks dangerous ;)

Reply:
Oh yeah; needed some help figuring that one out. It wasn't complaining before and all I did was "save" the function to a variable.

Reply:
Edit: made that harder than it had to be.